test/Confluent.SchemaRegistry.IntegrationTests/Tests/IsCompatible.cs

```diff
@@ -24,25 +23,25 @@ namespace Confluent.SchemaRegistry.IntegrationTests
     public static partial class Tests
     {
         [Theory, MemberData(nameof(SchemaRegistryParameters))]
-        public static void IsCompatible(Config config)
+        public static void IsCompatible_Topic(Config config)
         {
-            var topicName = Guid.NewGuid().ToString();
+            var sr = new CachedSchemaRegistryClient(new SchemaRegistryConfig { SchemaRegistryUrl = config.Server });
 
             var testSchema1 = 
                 "{\"type\":\"record\",\"name\":\"User\",\"namespace\":\"Confluent.Kafka.Examples.AvroSpecific" +
                 "\",\"fields\":[{\"name\":\"name\",\"type\":\"string\"},{\"name\":\"favorite_number\",\"type\":[\"i" +
                 "nt\",\"null\"]},{\"name\":\"favorite_color\",\"type\":[\"string\",\"null\"]}]}";
 
-            var sr = new CachedSchemaRegistryClient(new SchemaRegistryConfig { SchemaRegistryUrl = config.Server });
```

Comment:
This changes an existing test, possibly allowing for regression to slip through.
Instead extend or duplicate the test to cover both the old and new test cases.